bot/exts/evergreen/trivia_quiz.py

```diff
@@ -136,9 +359,14 @@ def check(m: discord.Message) -> bool:
                 # if number of hints sent or time alerts sent is less than 2, then send one.
                 if hint_no < 2:
                     hint_no += 1
+
                     if "hints" in question_dict:
                         hints = question_dict["hints"]
-                        await ctx.send(f"**Hint #{hint_no+1}\n**{hints[hint_no]}")
+
+                        await ctx.send(
```

Comment:
this again just seems like black formatting to its own specifications which arent necessarily better

Reply:
splitting it into two f-strings? that was from me, i thought it would be more understandable that way

Reply:
At this length (i.e. short) it just kinda spreads the code out, rather than helping very much with readability

Reply:
got it. resolved.